barretenberg/cpp/src/barretenberg/commitment_schemes/gemini/gemini.hpp

```diff
@@ -283,48 +337,6 @@ template <typename Curve> class GeminiVerifier_ {
 
         return batched_eval_accumulator;
     }
```

Comment:
The native branch of this code was moved to reduce verification to simplify my work. Basically GeminiVerifier is only going to be used for debugging, in the native universe, and this seemed the simplest approach